<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Aditya-Laptop" failures="1" tests="8" name="Default test" time="118.709" errors="0" timestamp="2023-05-12T21:52:51 IST">
  <testcase classname="com.project.skyfall.Brickworks_Website" name="TS001_VerifyProductNamePageTitle" time="0.042">
    <failure type="java.lang.AssertionError" message="expected [Brickworks Site Manager11] but found [Brickworks Site Manager]">
      <![CDATA[java.lang.AssertionError: expected [Brickworks Site Manager11] but found [Brickworks Site Manager]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.project.skyfall.Brickworks_Website.TS001_VerifyProductNamePageTitle(Brickworks_Website.java:76)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TS001_VerifyProductNamePageTitle -->
  <testcase classname="com.project.skyfall.Brickworks_Website" name="TS002_ValidUserLogin" time="0.296"/>
  <testcase classname="com.project.skyfall.Brickworks_Website" name="TS004_searchMetMap" time="9.5"/>
  <testcase classname="com.project.skyfall.Brickworks_Website" name="TS007_AddLocationLeftSideBar" time="23.207"/>
  <testcase classname="com.project.skyfall.Brickworks_Website" name="TS006_EditLocation" time="22.44"/>
  <testcase classname="com.project.skyfall.Brickworks_Website" name="TS005_CreateMarkerLocation" time="29.034"/>
  <testcase classname="com.project.skyfall.Brickworks_Website" name="TS008_deleteLocation" time="9.223"/>
  <testcase classname="com.project.skyfall.Brickworks_Website" name="TS009_UserLogOut" time="10.645"/>
  <testcase classname="com.project.skyfall.Brickworks_Website" name="TS003_InValidUserLogin" time="0.0">
    <ignored/>
  </testcase> <!-- TS003_InValidUserLogin -->
</testsuite> <!-- Default test -->
